extensions/yaml/package.json

```diff
@@ -13,6 +13,18 @@
   },
   "contributes": {
     "languages": [
+      {
+        "id": "compose",
```

Comment:
Does the order of these two definitions decide which language a doc gets assigned, since `docker-compose.yml` would match both `compose` and `yaml` language IDs?

Reply:
I confirmed that when compose is first then it gets picked correctly.

Reply:
We apparently also prioritize the pattern:

https://github.com/microsoft/vscode/blob/e8760a5d6c0e09ab692e5b3e414eca018745490f/src/vs/base/common/mime.ts#L194-L207